aws/scripts/sh/create_pr_comment.sh

```
@@ -0,0 +1,76 @@
+# Set up a cleanup action to log out of GitHub after the script completes
+trap 'gh auth logout' EXIT
+
+# Check if the script is running in the context of a pull request
+if [ "$IS_PULL_REQUEST" -eq 1 ]; then
+
+    echo "Running in the context of a pull request."
+
+    # Authenticate with GitHub using the token retrieved directly from AWS Secrets Manager
+    echo "Authenticating with GitHub..."
+
+    SECRET_ID=$(aws secretsmanager list-secrets --query 'SecretList[?contains(Name, `github-token`) && DeletedDate==null].Name' --output text)
```

Shellcheck (0.10.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)